### DISCUSSION TITLE
pymanopt with pytorch autograd

### DISCUSSION THREAD

**Discussion by Fengcheng-Pei** [cvxpy/cvxpy#2034] [Q&A]
Hello,

In my problem, there are no random settings. When I use pymanopt to solve the same problem with the help of pytorch autogard, the results are different. I just run the same code several times, but the results do not show the same number. I really can't understand and think that whether they are the tolerable computing errors?

Best regrads,
Fengcheng Pei

**Comment by phschiele** [comment]
@Fengcheng-Pei can you say a bit more about your problem? It's hard to judge which errors are tolerable without knowing the details. Ideal would be some code or program output.

**Reply by Fengcheng-Pei** [reply]
Hello,

Thanks great for your help! I solved the problem by adding the initial point.

Best regards,
Fengcheng Pei
